RecoLocalCalo/HcalRecAlgos/src/HcalDeterministicFit.cc

```diff
@@ -3,6 +3,10 @@
 #include <climits>
 #include "RecoLocalCalo/HcalRecAlgos/interface/HcalDeterministicFit.h"
 
+const float HcalDeterministicFit::invGpar[3];
+const float HcalDeterministicFit::negthre[2];
+const int HcalDeterministicFit::HcalRegion[2];
+
```

Comment:
The most common problem in CMSSW is attempting to take a reference of such thing, then it's odr-used. I have not looked at the code to exactly figure out the location where those data members could be odr-used.